GCAppirater.m

```diff
@@ -566,7 +482,8 @@ - (UIAlertController *)getEnjoyingAlertController {
                                                             style:UIAlertActionStyleDefault
                                                           handler:^(UIAlertAction *action) {
                                                             NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
-                                                            [userDefaults setBool:YES forKey:kGCAppiraterDeclinedToRate];
+                                                            [userDefaults setBool:YES forKey:kGCAppiraterUserDeclinedToRate];
+                                                            [userDefaults setObject:[NSDate distantFuture] forKey:kGCAppiraterAskAgainDate];
```

Comment:
My general feeling is that something like `distantFuture` isn't intended to be stored, but it'll probably be fine.